packages/main/src/methods/chat-session.ts

```diff
@@ -155,6 +165,16 @@ export class ChatSession {
             );
           }
         }
+      })
+      .catch((e) => {
+        // Errors in streamPromise are already catchable by the user as
```

Comment:
Did you mean to say `Errors in this._sendPromise...` here?

Reply:
No, streamPromise, the second step in the `this._sendPromise` chain. Everything after that isn't catchable by the user and is being handled and logged here.